src/app/app.config.ts

```diff
@@ -0,0 +1,8 @@
+import { ApplicationConfig } from '@angular/core';
+import { provideRouter } from '@angular/router';
+
+import { routes } from './app.routes';
+
```

Comment:
suggestion

          providers: [provideRouter(routes)],

          Explantion: It's a good practice to keep a comma after the last provider for consistency and future changes to the array.